Formula/bucklespring.rb

```diff
@@ -0,0 +1,25 @@
+class Bucklespring < Formula
+  desc "Nostalgia bucklespring keyboard sound"
+  homepage "https://github.com/zevv/bucklespring"
+  url "https://github.com/zevv/bucklespring/archive/v1.4.0.tar.gz"
+  sha256 "b38b2c0ec905573a243cb9db543f8eeb92e4d040025ef338d8bcfcfd5ccd5348"
+  license "GPL-2.0-only"
+
+  depends_on "make" => :build
+  depends_on "alure"
+  depends_on "pkg-config"
+
+  def install
+    system "
+      cp /usr/local/lib/pkgconfig/alure.pc mac/lib/pkgconfig/ ;
+      sed -i '' 's/-Wall -Werror/-Wall/' Makefile ;
+      PATH_AUDIO=#{prefix}/wav make
+    "
+    bin.install "buckle"
+    prefix.install "wav"
+  end
+
+  test do
+    system "buckle", "-h"
```

Comment:
Can you propose a more substantial test than checking the version or calling for help? These tests rarely ever fail even for broken installations.

>We want tests that don’t require any user input and test the basic functionality of the application. For example `foo build-foo input.foo` is a good test and (despite their widespread use) `foo --version` and `foo --help` are bad tests. 

Ref. https://docs.brew.sh/Formula-Cookbook#add-a-test-to-the-formula